app-box/app-box.html

```diff
@@ -135,14 +135,9 @@ <h2>Sub title</h2>
     Polymer({
       is: 'app-box',
 
-      behaviors: [
-        Polymer.AppScrollEffectsBehavior,
-        Polymer.IronResizableBehavior
-      ],
+      behaviors: [Polymer.AppScrollEffectsBehavior, Polymer.IronResizableBehavior],
```

Comment:
At what point does the formatter decide to have each item on its own line (`behaviors`, `listeners`, and `properties`)? This seems like it should be.

Reply:
It's using google styleguide, so I think it's after it hits 80 cols